src/main/java/eu/dissco/orchestration/backend/domain/ObjectType.java

```diff
@@ -1,20 +1,18 @@
 package eu.dissco.orchestration.backend.domain;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+
+@Getter
 public enum ObjectType {
-  SOURCE_SYSTEM ("sourceSystem", "https://hdl.handle.net/21.T11148/64396cf36b976ad08267"),
-  MAPPING ("mapping", "https://hdl.handle.net/21.T11148/b3f1045d8524d863ccfb"),
-  MAS("machineAnnotationService", "https://hdl.handle.net/21.T11148/64396cf36b976ad08267");
+  @JsonProperty("sourceSystem") SOURCE_SYSTEM ("https://hdl.handle.net/21.T11148/417a4f472f60f7974c12"),
```

Comment:
in the case of the orchestration service, it's a bit messier because we have 3 different objects, and i like coupling the Type with the profile in this case. 

Made some changes to the Fdo Service do a swtich statement on ObjectType. Let me know if you think that's better.